src/kernel/schedule.c

```diff
@@ -247,8 +249,6 @@ NOTRACE void __attribute__((noreturn)) runloop_internal()
                     ci->last_timer_update = here + runloop_timer_max;
                 }
             }
-           /* Make sure TLB entries are appropriately flushed before
-             * returning to userspace */
             page_invalidate_flush();
```

Comment:
Did you mean to also leave a call to page_invalidate_flush here?

Reply:
Yeah, that's not supposed to be there. Thanks!